PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -191,6 +201,10 @@ def miniAOD_customizeCommon(process):
     #
     ## PU JetID
     process.load("RecoJets.JetProducers.PileupJetID_cfi")
+    task.add(process.pileupJetId)
+    task.add(process.pileupJetIdCalculator)
+    task.add(process.pileupJetIdEvaluator)
```

Comment:
how was this task.add set of lines derived?
Can it come directly from PileupJetID_cfi in some way?
This file doesn't have to know what's in the imported file to reduce maintenance burden